.gitignore

```diff
@@ -5,3 +5,4 @@ lib/version.ts
 
 # Temporary directory for test execution
 tmp/
+node_modules
```

Comment:
I'd rather that this file only contain project-specific ignores and that stuff related to your developer environment be kept in your personal gitignore.

Reply:
Done. Sorry about that.

Reply:
Nothing to be sorry about. I might be abnormally strict about these things, but here I am benevolent. Also, maintaining this has made me tired of sugar coating things. This PR is an excellent starting point and the test is minimal yet it verifies desired behaviour, exactly how I like it.